src/sap/fhir/model/r4/FHIRListBinding.js

```diff
@@ -177,7 +177,7 @@ sap.ui.define([
 				// the direct next links will not be used by default to send the request
 				// instead its converted into the necessary parameters and path before sending
 				// this is to address the if the service url is relative
-				if (this.sNextLink && this.sNextLink.indexOf("?") > -1) {
+				if (this.sNextLink && this.sNextLink.indexOf("?") > -1 && this.oModel.bPreProcessNextLink) {
```

Comment:
do we really need the part "this.sNextLink.indexOf("?")"? AFAIK the next link could be any url, so why we should limit the url containing a "?"